exercises/atbash-cipher/cipher.t

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env perl6
+
+use Test;
+use JSON::Tiny;
+
+use lib ( my $dir = IO::Path.new($?FILE).parent ).path;
+
+my $module_name = %*ENV<EXERCISM>.so ?? 'Example' !! 'Allergies';
+my @potential_module = <p6 pm6 pm>.map:  $module_name ~ '.' ~ *; 
+
+my $module = first { $dir.child($_).e }, |@potential_module
+    or die "No file '$module_name.p6' found\n";
+
+require $module <&encode &decode>;
+
+plan 2;
+
+my %cases = from-json $dir.child('cases.json').slurp;
+
+subtest 'encode' => {
+    my @cases = |%cases.<encode>.<cases>;
+
+    plan +@cases;
+
+    is encode( .<phrase> ), .<expected>, .<description>
+        for @cases;
+};
+
+subtest 'decode' => {
+    my @cases = |%cases.<decode>.<cases>;
+
+    plan +@cases;
+
+    is decode( .<phrase> ), .<expected>, .<description>
+        for @cases;
+};
+
+done-testing;
```

Comment:
1. Having them both together results in `done-testing` doing nothing. The disadvantage of including code which does nothing is potential confusion for people reading or modifying said code. This is especially true for people doing exercises on the site who may be new to Perl 6. Furthermore, if someone modifies the code in future and changes the `plan` line, they may miss this line and commit changes without realising it has taken effect.

2. Re: being explicit, why have a function when you could have a comment which would definitely do nothing? Following that, would you see there being any advantage to leaving a comment at the end of the test?